modules/p2/ding_notifications/ding_notifications.module

```diff
@@ -28,3 +28,15 @@ function ding_notifications_clear_cache($account = NULL) {
 
   return cache_clear_all(DING_NOTIFICATIONS_CACHE_PREFIX . ':' . $account->uid, 'cache');
 }
+
+/**
+ * Implements hook_ding_user_frontend_build_menu_alter().
+ */
+function ding_notifications_ding_user_frontend_build_menu_alter(&$menu, $menu_item_defaults) {
+  $logout_item = array();
+  $logout_item = $menu_item_defaults;
+  $logout_item['#title'] = t('Notifications (@count)', array('@count' => ding_message_get_message_count()));
+  $logout_item['#href'] = '/user/me/view';
```

Comment:
It's a render array, using the `theme_menu_link`, so it's passed through the `l()` function later on in that theme.